docs/source/installation.rst

```diff
@@ -13,74 +19,69 @@ new environment by doing:
 	conda create -n eureka python==3.9.7
 	conda activate eureka
 
-
-Installation methods
---------------------
-
 a) With ``git`` and ``pip``
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Once in your new conda environment, you can install ``Eureka!`` directly from source on
 `GitHub <http://github.com/kevin218/Eureka>`_ using ``git`` and ``pip`` by running:
 
 .. code-block:: bash
 
-	git clone https://github.com/kevin218/Eureka.git
+	git clone -b v0.1-alpha https://github.com/kevin218/Eureka.git
 	cd Eureka
 	pip install -e '.[jwst]'
 
-To update your ``Eureka!`` installation to the most recent version, you can do the following within that Eureka folder
+To update your ``Eureka!`` installation to the most recent version, you can do the following within that Eureka folder:
 
 .. code-block:: bash
 
 	git pull
 	pip install --upgrade '.[jwst]'
 
-b) With ``git`` and ``conda``
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+b) With ``pip`` only
+~~~~~~~~~~~~~~~~~~~~
 
-Once in your new conda environment, you can install ``Eureka!`` using ``git`` and ``conda``. This can be done following:
+Once in your new conda environment, you can install the ``Eureka!`` package with ``pip`` with the following command:
 
 .. code-block:: bash
 
-	git clone https://github.com/kevin218/Eureka.git
-	cd Eureka
-	conda env create --file environment.yml --force
-	conda activate eureka
-	pip install --no-deps .
+	pip install -e git+https://github.com/kevin218/Eureka.git@v0.1-alpha#egg=eureka[jwst]
 
-To update your ``Eureka!`` installation to the most recent version, you can do the following within that Eureka folder
+Other specific branches can be installed using:
 
 .. code-block:: bash
+	
+	pip install -e git+https://github.com/kevin218/Eureka.git@mybranchname#egg=eureka[jwst]
 
-	git pull
-	conda env update --file environment.yml --prune
-	pip install --no-deps --upgrade .
+If you desire any of the files in the `demos folder <https://github.com/kevin218/Eureka/tree/main/demos>`_, you will have to download these from
+GitHub following the method described below.
 
+To update your ``Eureka!`` installation to the most recent version, you can then run the following command:
 
-c) With ``pip``
-~~~~~~~~~~~~~~~
+.. code-block:: bash
 
-Once in your new conda environment, you can install the ``Eureka!`` package with ``pip`` with the following command:
+	pip install --upgrade git+https://github.com/kevin218/Eureka.git#egg=eureka[jwst]
 
-.. code-block:: bash
 
-	pip install -e git+https://github.com/kevin218/Eureka.git#egg=eureka[jwst]
+Setting up a ``conda`` environment directly from the ``git`` repository
```

Comment:
I would name this something like "Installing with a conda environment.yml file" instead